crypto/evp/e_aes.c

```diff
@@ -2584,14 +2584,22 @@ static int aes_gcm_ctrl(EVP_CIPHER_CTX *c, int type, int arg, void *ptr)
         memcpy(ptr, c->buf, arg);
         return 1;
 
+    case EVP_CTRL_AEAD_GET_TAGLEN:
```

Comment:
EVP_CIPHER_CTX_tag_length() has been added - so this should not be necessary.

Reply:
This is no longer used so marking as resolved.